src/Marten.Testing/Marten.Testing.csproj

```diff
@@ -28,7 +28,7 @@
     <ProjectReference Include="..\Marten.Testing.OtherAssembly\Marten.Testing.OtherAssembly.csproj" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="Jil" Version="2.17.0" />
+    <PackageReference Include="Jil" Version="3.0.0-alpha2" />
```

Comment:
**CONCERN:** I don't' think that we should not use prerelease version (even just for the samples). @lhaussknecht Is it needed for samples updated?

Reply:
@oskardudycz . NET Framework 4.6.1 is Not supported in the older version of Jil. Only the latest pre-release version supports it. So I had made changes to the original PR to use this version so that the code compiles. Note that the code was earlier under a conditional if block so it was not even compiling.

Reply:
@mysticmind thank you for clarification. After reconsidering - I'm fine with this change.